### PR TITLE
Rename cli content view tests as per issue #3083

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: tests/foreman/cli/test_contentviewfilter.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def setUpClass(cls):
         })
 
     @tier1
-    def test_create_cvf_different_names(self):
+    def test_positive_create_with_name_by_cv_id(self):
```
Comment (sghai) [inline tests/foreman/cli/test_contentviewfilter.py]: `test_positive_create_with_name_attach_cv_by_id` ?

File: tests/foreman/cli/test_contentviewfilter.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def test_create_cvf_different_names(self):
                 self.assertEqual(cvf['type'], filter_content_type)
 
     @tier1
-    def test_create_cvf_content_types(self):
+    def test_positive_create_with_content_type_by_cv_id(self):
```
Comment (sghai) [inline tests/foreman/cli/test_contentviewfilter.py]: `test_positive_create_with_content_type_attach_cv_by_id`?

File: tests/foreman/cli/test_contentviewfilter.py [hunk 3]
```diff
@@ -145,7 +145,7 @@ def test_create_cvf_description(self):
         self.assertEqual(cvf['description'], description)
 
     @tier1
-    def test_create_cvf_by_cv_name(self):
+    def test_positive_create_by_cv_name(self):
```
Comment (sghai) [inline tests/foreman/cli/test_contentviewfilter.py]: `test_positive_create_assign_cv_by_name` ?

File: tests/foreman/cli/test_contentviewfilter.py [hunk 4]
```diff
@@ -120,7 +120,7 @@ def test_create_cvf_inclusions(self):
 
     @tier1
     @skip_if_bug_open('bugzilla', 1236532)
-    def test_create_cvf_description(self):
+    def test_positive_create_with_description_by_cv_id(self):
```
Comment (sghai) [inline tests/foreman/cli/test_contentviewfilter.py]: `test_positive_create_with_description_assign_cv_by_id` ?

File: tests/foreman/cli/test_contentviewfilter.py (hunk 3 above)
Comment (sghai) [inline tests/foreman/cli/test_contentviewfilter.py]: If you like this naming convention then please update other test names accordingly. thank you.

**Comment by sghai** [comment]
ACK.. Few non-blocking comments.

File: tests/foreman/cli/test_contentviewfilter.py (hunk 1 above)
Comment (abalakh) [inline tests/foreman/cli/test_contentviewfilter.py]: imo `test_positive_create_with_name_by_cv_id` is shorter and still makes sence. Also this is the approach we use almost everywhere in tests names, so i'd vote for leaving it as it is. Same for `assign`.

File: tests/foreman/cli/test_contentviewfilter.py [hunk 5]
```diff
@@ -334,7 +334,7 @@ def test_create_cvf_repos_docker(self):
             self.assertIn(repo['id'], repos)
 
     @tier1
-    def test_create_cvf_names_negative(self):
+    def test_negative_create_with_name(self):
```
Comment (abalakh) [inline tests/foreman/cli/test_contentviewfilter.py]: `test_negative_create_with_invalid_name`? We used such approach for UI/API tests. I know you don't like it, `negative` and `invalid` share similar purpose, so it's not a blocker :)

Comment (abalakh) [inline tests/foreman/cli/test_contentviewfilter.py]: Basically, `test_negative_create_with_invalid_name` fits `test_negative_create_with_same_name` better, but that's just imo.

**Comment by abalakh** [comment]
ACK, one small non-blocking suggestion.

File: tests/foreman/cli/test_contentviewfilter.py (hunk 1 above)
Comment (sthirugn) [inline tests/foreman/cli/test_contentviewfilter.py]: @sghai - @abalakh is right, we are following the naming convention from https://github.com/SatelliteQE/robottelo/issues/3083#issuecomment-161375918.  And use of additional verbs like `attach` or `assign` is not required if it makes sense without them.

File: tests/foreman/cli/test_contentviewfilter.py (hunk 5 above)
Comment (sthirugn) [inline tests/foreman/cli/test_contentviewfilter.py]: ty @abalakh, will update this to `test_negative_create_with_invalid_name`

File: tests/foreman/cli/test_contentviewfilter.py (hunk 2 above)
Comment (sthirugn) [inline tests/foreman/cli/test_contentviewfilter.py]: same response as above

File: tests/foreman/cli/test_contentviewfilter.py (hunk 3 above)
Comment (sthirugn) [inline tests/foreman/cli/test_contentviewfilter.py]: same response as above

File: tests/foreman/cli/test_contentviewfilter.py (hunk 4 above)
Comment (sthirugn) [inline tests/foreman/cli/test_contentviewfilter.py]: same response as above

**Comment by sthirugn** [comment]
@abalakh @sghai comments addressed and rebased

File: tests/foreman/cli/test_contentviewfilter.py (hunk 1 above)
Comment (sghai) [inline tests/foreman/cli/test_contentviewfilter.py]: :+1:

**Comment by sghai** [comment]
ACK

**Comment by abalakh** [comment]
APT

**Pull request merged**